versions.tf

```
@@ -1,11 +1,15 @@
 terraform {
-  required_version = ">= 1.0"
+  required_version = "> 1.0"
```

The `nullable` feature was introduced in 1.1. I can't keep track of which features we are using and when they were introduced, so I just set the minimum version to 1.3, which I know covers just about everything, but it is up to you. At least set it to 1.1 since we know 1.0 will not work.

```suggestion
  required_version = ">= 1.1"
```